testsupport/wait/http_request.go

```diff
@@ -0,0 +1,211 @@
+package wait
```

Comment:
I believe it would be better located in the `testsupport` package. The `wait` package is for func with wait/poll logic, so reading `wait.NewHTTPRequest()` is a bit confusing :(

Reply:
I think I will need to create a new package, since `testsupport` uses `wait` and `NewHTTPRequest` is being used in host.go by `WaitForUserSignupReadyInRegistrationService`, this will create a circular dependency error.

Any suggestion for the new package name or an existing one (different then `testsupport`) ?